pkg/loader/load.go

```diff
@@ -206,6 +206,14 @@ func (s *loaderImpl) Close() {
 var utilGetTableInfo = getTableInfo
 
 func (s *loaderImpl) refreshTableInfo(schema string, table string) (info *tableInfo, err error) {
+	if len(schema) == 0 {
```

Comment:
why schema would be empty? what's the case?

Reply:
user may forget set the DDL.Database